### ISSUE TITLE
Update WIKI with technical details: Login

### ISSUE THREAD

**Issue by maulieshah** [hackforla/tdm-calculator#1861]
### Overview
We need to add technical details for how the Login page works so that the stakeholder's new vendor or in house development team can take over the project.

#### Details
This page also serves as an information source for development issues while the site is under development by Hack for LA.

### Action Items
- [x] Add Technical Details

#### WIKI pages
- https://github.com/hackforla/tdm-calculator/wiki/Login
- https://github.com/hackforla/tdm-calculator/wiki/Registration-and-Login
- https://github.com/hackforla/tdm-calculator/wiki/Update-Account
- https://github.com/hackforla/tdm-calculator/wiki/Forgot-Password

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - [Staging Deck, slide 1861](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?pli=1#slide=id.g3389954eb7d_9_24)
- Release Deck
   - [YYYY-MM-DD Release Deck, slide 1861]

**Comment by ExperimentsInHonesty** [comment]
@del9ra Please provide update.  I wouldn't normally ask this soon, but since its only a 0.5pt issue... 

<details><summary>Instructions</summary>

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."
</details> 

1. Progress: 
2. Blockers: 
3. Availability:
4. ETA:
5. Pictures (if necessary):

**Comment by del9ra** [comment]
@ExperimentsInHonesty You're right, Bonnie. I forgot to update. I'll make sure to provide more regular updates going forward.

Progress: Added technical details to the login page https://github.com/hackforla/tdm-calculator/wiki/Login
Blockers: no blockers
Availability: Monday-Friday 1pm-3pm
ETA: Completed

**Comment by kcoronel** [comment]
@ExperimentsInHonesty I added this as a subtask to #1485 and it looks good and ready to close

**Comment by entrotech** [comment]
The wiki page https://github.com/hackforla/tdm-calculator/wiki/Update-Account is still missing the Technical Details section.

**Comment by ExperimentsInHonesty** [comment]
@del9ra John has reviewed the issue and left a note in a comment above

**Comment by del9ra** [comment]
@entrotech Hi John, 
but I didn't assign myself to adding technical details to the Update account. I only assigned myself to the Login page issue.

**Comment by del9ra** [comment]
@ExperimentsInHonesty Okay, that makes sense. Thanks!
